x-pack/plugins/ml/public/application/components/custom_urls/custom_url_editor/utils.ts

```diff
@@ -280,7 +352,8 @@ function buildAppStateQueryParam(queryFieldNames) {
 // Builds the full URL for testing out a custom URL configuration, which
 // may contain dollar delimited partition / influencer entity tokens and
 // drilldown time range settings.
-export async function getTestUrl(job, customUrl) {
+async function getAnomalyDetectionJobTestUrl(job: Job, customUrl: UrlConfig): Promise<string> {
+  // @ts-ignore
```

Comment:
can we get rid of this `@ts-ignore`?

Reply:
Removed in 642ce6dbe70bb719b6416e5a61184a6b966b71d7